deploy-service/teletraanservice/pom.xml

```diff
@@ -37,6 +38,11 @@
             <artifactId>dropwizard-jackson</artifactId>
             <version>${dropwizard.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Why is the dependency also added here? Is it a maven requirement?

Reply:
Good catch. I intend to use here as well but plan changed to start with those simple counters in common first.